tests/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -1042,6 +1042,17 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
       Type.Param(Nil, Type.Name("T"), Nil, Type.Bounds(None, Some(Type.Name("Int"))), Nil, Nil)
     )
     assertEquals(tree.syntax, "T <: Int")
+
+    assertTree(templStat("def foo[T <: Int] = ???")) {
+      Defn.Def(
+        Nil,
+        Term.Name("foo"),
+        tree :: Nil,
```

Comment:
```suggestion
       Type.ParamClause(tree :: Nil),
```
this apply also gets generated? Should we only use the new one in added tests?

Reply:
sure, can do that, add clause tree for new tests explicitly.

Reply:
tests done.